test/feature-benchmark/mzcompose.py

```diff
@@ -81,7 +81,7 @@ def make_comparator(name: str) -> Comparator:
     return RelativeThresholdComparator(name, threshold=0.10)
 
 
-default_timeout = "30m"
+default_timeout = "1800s"
```

Comment:
?
Edit: Ok, I see that we are passing it to materialize_params now too. I guess it doesn't support `30m`

Reply:
Yes, they are apparently parsed in a different ways.